README.md

```diff
@@ -1,2 +1,52 @@
 # Overview
-This is a healhcare portal
+This is a healhcare portal (hospital portal) which enables the patients to easily access medical attention from the respective healthcare facility. The application 'on full functionality' will emable booking of the medical services, store and manage patient data (such as medical history, lab results).
+
+### Important Note
+This is a general application that must be cstomized to be used by the sourcing organization
+ 
+## Built With
+
+- HTML5
+- CSS3
+- JavaScript (ES6)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- web browser of choice is installed and configured
+
+### Usage
+ - This is a sample or test web application, with limited or no functionality
+
+### Deployment
+ - open the index.html page using any browser (preferably - firefox, googlechrome, safari)
+
+## Authors
+
+👤 **Calvince**
+
+- GitHub: [@githubhandle](https://github.com/calvnce)
+- Twitter: [@twitterhandle](https://twitter.com/sir_coly)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/sir_coly)
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone who helped complete the project
+
```

Comment:
- [ ] Instead of writing this. When you update your design, You should mention particularly by writing or giving link. (Hint: 
[Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015))

Reply:
:heavy_check_mark: Well Done